src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -29,6 +29,7 @@
     private Set<String> eventTypes;
 
     @NotNull
+    @Size(min = 1, message = "must contain at least one character")
```

Comment:
For me 1 and 2 chars can also make sense. And 3 chars is not a guarantee that it will make sense :)
So I would prefer to keep it as it is.